src/lib.rs.in

```diff
@@ -25,15 +22,17 @@ use serde_json::builder::ObjectBuilder;
 use serde_json::de::from_reader;
 use serde_json::value::Value;
 use rand::{OsRng, Rng};
-use redis::Client;
 use rustc_serialize::base64::{self, ToBase64};
-use std::collections::BTreeMap;
+use std::collections::HashMap;
```

Comment:
I've prepared this in a separate commit, d516846edc4. Please rebase on top of that.